File: src/main/java/org/arkadst/tempmotd/SetMOTDCommand.java
package org.arkadst.tempmotd;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.jetbrains.annotations.NotNull;

import java.io.FileWriter;
import java.io.IOException;

public class SetMOTDCommand implements CommandExecutor {

    Main main;
    public SetMOTDCommand(Main main){
        this.main = main;
    }

    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {

        if (args.length >= 1){
            StringBuilder message = new StringBuilder();
            for (int x = 0; x < args.length; x++){
                message.append(args[x]);
                if (x != args.length - 1){
                    message.append(" ");
                }
            }
            Main.MOTD = message.toString();

            try {
                FileWriter isOp_file_writer = new FileWriter(main.motdtxt);
                isOp_file_writer.write(Main.MOTD);
                isOp_file_writer.close();
            } catch (IOException e){
                e.printStackTrace();
            }

        } else {
            Main.MOTD = "";
            main.motdtxt.delete();
        }
        return true;
    }
}

File: src/main/kotlin/org/arkadst/tempmotd/SetMOTDCommand.kt
package org.arkadst.tempmotd

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import java.lang.StringBuilder
import java.lang.NumberFormatException

class SetMOTDCommand(private val main: Main) : CommandExecutor {

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        val motd = main.config.getStringList("motd")
        if (args.isNotEmpty()) {
            val message = StringBuilder()
            for (x in 1 until args.size) {
                message.append(args[x])
                if (x != args.size - 1) {
                    message.append(" ")
                }
            }
            try {
                val row = args[0].toInt()
                if (row == 1 || row == 2) {
                    motd[row - 1] = message.toString()
                    main.config.set("motd", motd)
                    main.saveConfig()
                    main.reloadConfig()
                } else {
                    return false
                }
            } catch (e: NumberFormatException) {
                return false
            }
        } else {
            return false
        }
        return true
    }
}